test/FsAutoComplete.Tests.Lsp/CodeFixTests/IgnoreExpressionTests.fs

```diff
@@ -0,0 +1,61 @@
+module private FsAutoComplete.Tests.CodeFixTests.IgnoreExpressionTests
+
+open Expecto
+open Helpers
+open Utils.ServerTests
+open Utils.CursorbasedTests
+open FsAutoComplete.CodeFix
+
+let tests state =
+  serverTestList (nameof IgnoreExpression) state defaultConfigDto None (fun server ->
+    [ let selectCodeFix = CodeFix.withTitle IgnoreExpression.title
+
+      testCaseAsync "ignore constant"
+      <| CodeFix.check
+        server
+        "
+let a b =
+    9$0
+    null"
+        Diagnostics.acceptAll
+        selectCodeFix
+        "
+let a b =
+    9 |> ignore
```

Comment:
Would it be worth adding one test where parens _do_ need to be added as a sanity check? Something like an unparenthesized tuple should work, e.g.,

```fsharp
1, 2
```

→

```fsharp
(1, 2) |> ignore
```

Reply:
Good idea, add a test.